Git Formats/Git Config.sublime-syntax

```diff
@@ -115,11 +115,10 @@ contexts:
   # changed = red
   # untracked = bold green
   key-color-pair:
-    - meta_scope: meta.section.body.git.config
-    - match: ^(\s*)({{variable_name}})(\s*(\=)\s*)
+    - match: ^(\s*)({{key_name}})(\s*(=)\s*)
       captures:
         1: meta.mapping.git.config
-        2: meta.mapping.key.git.config variable.other.readwrite.git.config
+        2: meta.mapping.key.git.config string.unquoted.git.config
```

Comment:
I really find this to be a negative change. I could tweak my color scheme to compensate, but it's going to be a downside for most people.

Reply:
I have added an override in my color scheme to make `meta.mapping.key string` a distinct color. We'll see if I hate it in some other context like, say, Python.

Who knows, maybe it'll be better for dict keys there, too.

Reply:
That's a general issue JSON and YAML suffer from as well.

see: https://github.com/sublimehq/sublime_text/issues/3590

Reply:
Yeah. I forgot I used to have an override on the old computer.